app/controllers/api/v1/medical_shifts_controller.rb

```diff
@@ -3,9 +3,18 @@
 module Api
   module V1
     class MedicalShiftsController < ApiController
-      after_action :verify_authorized, except: :index
+      after_action :verify_authorized, except: [:index, :hospital_name_suggestion_index]
       after_action :verify_policy_scoped, only: :index
 
+      def hospital_name_suggestion_index
+        result = MedicalShifts::HospitalNameSuggestion.call
```

Comment:
Hey, this method should be something like the code below. We want to return only the suggestions for the hospital_names used by the current user.
```
authorized_scope = policy_scope(MedicalShift)
        medical_shifts = MedicalShifts::HospitalNameSuggestion.result(
         scope: authorized_scope,
       ).medical_shifts
```

Reply:
@ZeFelix , please check if the method `MedicalShiftPolicy#index?` Is being used or not, if it is not, please remove it.
```
 def index?
    user.present?
  end
```